### DISCUSSION TITLE
How can I be a contributor for this project?

### DISCUSSION THREAD

**Discussion by dumihi313** [Abhinandan-Kushwaha/react-native-gifted-charts#780] [Q&A]
I see some contributors to your project
![image](https://github.com/user-attachments/assets/416c7067-482a-48aa-8249-965f893fb98d)

I am eager to be one of them on this list, how to be a part of this list ;)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @dumihi313 
I am glad to know you want to contribute.
Here is the [contribution guide](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/CONTRIBUTING.md)
You can fix an issue you are facing, or any other [open issue](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/issues)

**Reply by jeanniton-mnr** [reply]
> "Please do NOT format the files. We are trying to maintain a unified syntax and keep the reviewing process fast and simple. - [contribution guide](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/CONTRIBUTING.md)"

While there are only 4 open PR, would it not be ideal to format the code so it is more readable after you have approved those PRs?
Then, in the future, every PR will use and take advantage of the formatted code.
